motr/io_nw_xfer.c

```diff
@@ -975,10 +991,24 @@ static void target_ioreq_calc_idx(struct m0_op_io *ioo,
 				&fop_cs_data->cd_idx[fop_cs_data->cd_num_units];
 			M0_ASSERT(cs_idx->ci_pg_idx == UINT32_MAX &&
```

Comment:
I have a question here:
ci_pg_idx is declared as uint64_t.
Why are we comparing it against UINT32_MAX, but not UINT64_MAX?

Reply:
in ioreq_fop_checksum_data_init, it looks to have initialized with UINT32_MAX instead of UINT64_MAX;  will correct it during  next work around DI